### PR TITLE
Gmail&Zendesk

### PR DESCRIPTION
**Pull request by ashalya**
Pom changed

### CODE REVIEW

File: zendesk/zendesk-connector/zendesk-connector-1.0.0/org.wso2.carbon.connector/pom.xml [hunk 1]
```diff
@@ -134,7 +122,7 @@
 						<sec.verifier.dir>${basedir}/target/security-verifier/</sec.verifier.dir>
 						<maven.test.haltafterfailure>false</maven.test.haltafterfailure>
 						<carbon.zip>
-							${basedir}/repository/wso2esb-${esb.version}.zip
+                            ${basedir}/../../../../repository/wso2esb-${esb.version}.zip
```
Comment (Shakila) [inline zendesk/zendesk-connector/zendesk-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: Format this pom

File: gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml [hunk 2]
```diff
@@ -2,7 +2,12 @@
 <project
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
 	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-
+    <parent>
+        <groupId>org.wso2.carbon</groupId>
+        <artifactId>esb-connector-parent</artifactId>
+        <version>4.3.0-SNAPSHOT</version>
+        <relativePath>../../../../pom.xml</relativePath>
+    </parent>
```
Comment (Shakila) [inline gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml]: Reformat this pom

File: gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml [hunk 3]
```diff
@@ -2,7 +2,12 @@
 <project
```
Comment (Shakila) [inline gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml]: Add license header

File: gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml (hunk 2 above)
Comment (ashalya) [inline gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml]: fixed in https://github.com/wso2/esb-connectors/pull/432

File: gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml (hunk 3 above)
Comment (ashalya) [inline gmail/gmail-connector/gmail-connector-1.0.0/gmail/pom.xml]: fixed in https://github.com/wso2/esb-connectors/pull/432

File: zendesk/zendesk-connector/zendesk-connector-1.0.0/org.wso2.carbon.connector/pom.xml (hunk 1 above)
Comment (ashalya) [inline zendesk/zendesk-connector/zendesk-connector-1.0.0/org.wso2.carbon.connector/pom.xml]: fixed in https://github.com/wso2/esb-connectors/pull/432

**Comment by ashalya** [comment]
licence  header changed in https://github.com/wso2/esb-connectors/pull/449

**Pull request merged**